src/components/DataTable/__mocks__/mockPaginationTable.js

```diff
@@ -0,0 +1,499 @@
+      return (
+        <Table.SelectionHeading aria-label={column.header.label} {...cellProps}>
+          <ControlLabel srOnly htmlFor="selectAll">
+            Select all rows
+          </ControlLabel>
+          <input
+            type="checkbox"
+            id="selectAll"
+            checked={!unselectedRows}
+            onChange={this.onSelectAllRows}
+          />
+        </Table.SelectionHeading>
+      );
+    };
+
+    // action header cell custom formatter
+    const actionHeaderCell = ({ column, cellProps }) => {
+      return (
+        <Table.Heading aria-label={column.header.label} {...cellProps}>
+          {column.header.label}
+        </Table.Heading>
+      );
+    };
+
+    // cell formatter
+    const cellFormat = value => {
+      return <Table.Cell>{value}</Table.Cell>;
+    };
+
+    // selection cell formatter
+    const selectionCellFormat = (value, { rowData, rowIndex }) => {
+      const id = `select${rowIndex}`;
+      const label = `Select row ${rowIndex}`;
+      return (
+        <Table.SelectionCell>
+          <ControlLabel srOnly htmlFor={id}>
+            {label}
+          </ControlLabel>
+          <input
+            type="checkbox"
+            id={id}
+            checked={rowData.selected}
+            onChange={e => {
+              this.onSelectRow(e, rowData);
+            }}
+          />
+        </Table.SelectionCell>
+      );
+    };
+
+    this.state = {
+      // Sort the first column in an ascending way by default.
+      sortingColumns: {
+        name: {
+          direction: 'asc',
+          position: 0
+        }
+      },
+
+      // column definitions
+      columns: [
+        {
+          property: 'select',
+          header: {
+            label: 'Select all rows',
+            props: {
+              index: 0,
+              rowSpan: 1,
+              colSpan: 1
+            },
+            customFormatters: [selectionHeaderCell]
+          },
+          cell: {
+            props: {
+              index: 0
+            },
+            formatters: [selectionCellFormat]
+          }
+        },
+        {
+          property: 'name',
+          header: {
+            label: 'Name',
+            props: {
+              index: 1,
+              rowSpan: 1,
+              colSpan: 1
+            },
+            transforms: [sortable],
+            formatters: [sortingFormat],
+            customFormatters: [sortableHeaderCell]
+          },
+          cell: {
+            props: {
+              index: 1
+            },
+            formatters: [cellFormat]
+          }
+        },
+        {
+          property: 'height',
+          header: {
+            label: 'Height',
+            props: {
+              index: 2,
+              rowSpan: 1,
+              colSpan: 1
+            },
+            transforms: [sortable],
+            formatters: [sortingFormat],
+            customFormatters: [sortableHeaderCell]
```

Comment:
`customFormatters` is only due to the fact that reactabular sortabular formatters do not appear to support wrapped formatters. I will follow up with @bebraw on this, as ideally, we can just support `formatters: [sortingFormat, sortableHeaderCell]`. I've mimicked this closely using my own custom formatter in addition to reactabular formatters for now.